platforms/documentation/docs/src/docs/userguide/authoring-builds/basics/plugins.adoc

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-[[plugins]]
```

Comment:
Keep both anchors?

Reply:
This was a duplicate anchor. All the anchors removed are duplicates and the ones that are necessary I renamed.